README.md

```diff
@@ -0,0 +1,26 @@
+## Setup
+
+### install docker
+Installation instructions for [debian/linux](https://docs.docker.com/engine/installation/linux/docker-ce/debian/), [macOS](https://docs.docker.com/docker-for-mac/install/) and [windows](https://docs.docker.com/docker-for-windows/install/)
```

Comment:
If we mention Debian explicitly here and as we have a couple of Arch-Users on the Treff, we should add `pacman -S docker` as the installation command for Arch. It's in the default repos, no need to link somewhere.